quick_blue_linux/pubspec.yaml

```diff
@@ -11,6 +11,8 @@ environment:
 dependencies:
   flutter:
     sdk: flutter
+  bluez: ^0.7.8
+  rxdart: ^0.27.3
```

Comment:
I would not recommand `rxdart` as Dart has Future/Stream already

Reply:
yeah , i was checking few things from RxDart for that stituation i explained earlier , but seems like a simple timer will do the job